index.js

```diff
@@ -150,11 +150,15 @@ function mungeCmd (workingDir, options) {
 
   var m = command.match(re)
   var replace
+  var nodeCmd = workingDir + '/node.cmd'
+  if(workingDir.indexOf(' ') !== -1) {
+    nodeCmd = ' "' + nodeCmd + '"'
```

Comment:
any reason to not just always wrap the `nodeCmd` in quotes for Windows?

Reply:
Correct. I don't think there are side-effects.